src/nile/cli.py

```diff
@@ -254,32 +261,32 @@ def version():
 @click.argument("tx_hash", nargs=1)
 @network_option
 @click.option("--contracts_file", nargs=1)
-def debug(tx_hash, network, contracts_file):
+async def debug(tx_hash, network, contracts_file):
     """Locate an error in a transaction using contracts."""
-    debug_command(normalize_number(tx_hash), network, contracts_file)
+    await debug_command(normalize_number(tx_hash), network, contracts_file)
 
 
 @cli.command()
 @network_option
 @click.option("--predeployed/--registered", default=False)
-def get_accounts(network, predeployed):
+async def get_accounts(network, predeployed):
     """Retrieve and manage deployed accounts."""
     if not predeployed:
-        return get_accounts_command(network)
+        return await get_accounts_command(network)
     else:
-        return get_predeployed_accounts_command(network)
+        return await get_predeployed_accounts_command(network)
 
 
 @cli.command()
 @click.argument("contract_address")
 @network_option
-def get_nonce(contract_address, network):
+async def get_nonce(contract_address, network):
     """Retrieve the nonce for a contract."""
-    return get_nonce_command(normalize_number(contract_address), network)
+    return await get_nonce_command(normalize_number(contract_address), network)
 
 
 cli = load_plugins(cli)
 
 
 if __name__ == "__main__":
-    cli()
+    cli(_anyion_backend="asyncio")
```

Comment:
this results in this being logged every time:
```
Using selector: EpollSelector
```

<img width="1071" alt="image" src="https://user-images.githubusercontent.com/1703831/200955469-fe282c25-f2ce-435a-8c3a-bad91b6692ae.png">


try [disabling the logging](https://stackoverflow.com/a/60503706/1633783)